### ISSUE TITLE
7.1.18. Screen Name/Title: Creating an account - Lack of Explicit Form Grouping

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2411]
Component: Creating an account form
Observation:
Lack of Explicit Form Grouping 
Input fields are not grouped using <fieldset> and <legend> where grouping is semantically useful.

(Attachment: 7.1.18. Lack of Explicit Form Grouping.png)

Recommendation: 
Recommendation: Consider using <fieldset> and <legend> for logically grouped fields (e.g., contact info) to improve accessibility for screen reader users.
